### PR TITLE
[GCI-142] Delete redundant Collection.addAll() call (replaced with constructor)

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Issue: https://issues.openmrs.org/browse/GCI-142
Related PR: #2366

I've replaced Collection.addAll() call with passing collection's content to it's constructor

I didn't find addresses Sonar issue, but IntelliJ Inspector gave me that hint.
I hope it's redundant to do this:
`List<> list = new ArrayList<>();
list.addAll(someCollection);`

Rather, I would do it just in List constructor

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you give a direct link to the sonar defect that you are trying to address?

**Comment by BartlomiejRasztabiga** [comment]
Some of issues that I wanted to fix aren't only Sonar or Findbugs issues. But I hope they also improve code quality :)

**Comment by dkayiwa** [comment]
Then you need to explain them more and how they improve code quality.

**Comment by BartlomiejRasztabiga** [comment]
Of course I will, I'm currently working on it :) I think I will finish it by today.

**Comment by BartlomiejRasztabiga** [comment]
@dkayiwa 
I've edited PR description :)

**Comment by dkayiwa** [comment]
Can you look into these merge conflicts?

**Comment by BartlomiejRasztabiga** [comment]
@dkayiwa Fixed :)

File: api/src/test/java/org/openmrs/api/OrderSetServiceTest.java [hunk 1]
```diff
@@ -348,9 +348,8 @@ private OrderSet orderSetBuilder(boolean orderSetRetired, boolean orderSetMember
 		orderSetMember.setDateCreated(new Date());
 		orderSetMember.setRetired(orderSetMemberRetired);
 		orderSetMember.setOrderSet(orderSet);
-		
-		List<OrderSetMember> orderSetMembers = new ArrayList<>();
-		orderSetMembers.addAll(Collections.singletonList(orderSetMember));
+
+		List<OrderSetMember> orderSetMembers = new ArrayList<>(Arrays.asList(orderSetMember));
```
Comment (dkayiwa) [inline api/src/test/java/org/openmrs/api/OrderSetServiceTest.java]: Did you intentionally change this from Collections.singletonList?

Comment (BartlomiejRasztabiga) [reply api/src/test/java/org/openmrs/api/OrderSetServiceTest.java]: By accident while resolving conflicts. Fixed already, thanks :)

**Comment by dkayiwa** [comment]
Thanks @BartlomiejRasztabiga 😊

**Pull request merged**